projects/angular-jwt/src/lib/jwt.interceptor.ts

```diff
@@ -40,32 +40,38 @@ export class JwtInterceptor implements HttpInterceptor {
 
   isWhitelistedDomain(request: HttpRequest<any>): boolean {
     const requestUrl: any = parse(request.url, false, true);
-
+    console.log(requestUrl);
     return (
       requestUrl.host === null ||
-      this.whitelistedDomains.findIndex(
-        domain =>
-          typeof domain === 'string'
-            ? domain === requestUrl.host
-            : domain instanceof RegExp
-              ? domain.test(requestUrl.host)
-              : false
+      this.whitelistedDomains.findIndex((domain) =>
+        typeof domain === "string"
+          ? domain === requestUrl.hostname
```

Comment:
If we're now checking `hostname` here, should the check on line 45 change to `requestUrl.hostname` to match?